packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -550,7 +550,6 @@ export default {
 #upload-info {
   background-color: var(--oc-color-background-secondary);
   width: 400px;
-  z-index: 4;
```

Comment:
fyi: z-index has been removed here because it's sufficient that the `snackbar` div in the `Application` layout has a high z-index. The notifications that get rendered into the sidebar have their own z-index again, which is not strictly necessary. Didn't want to make an ODS PR for that though.